pkg/server/server.go

```diff
@@ -56,6 +58,25 @@ type Config struct {
 
 	// Array of profiles names that will be generated on each profiling tick.
 	ProfilingNames []string
+
+	// Backdate is how long to backdate signed identities to account for clock
+	// skew.
+	Backdate time.Duration
```

Comment:
I'd be happy to hardcode this to some reasonable value... the amount of config that has been pulled up as a result of these changes feels like a little bit of a smell

Reply:
what configuration values would you like hardcoded? just backdate? pulling out code from serverca into core necessitated core configuration changes, i'd be happy for suggestions on alternate approaches.